src/bio.c

```diff
@@ -61,18 +61,13 @@
 #include "server.h"
 #include "bio.h"
 
+#define MASK_UNUSED_API
```

Comment:
I would rather just delete the unused code, it hasn't been used in 7 years as far as I can tell, and it'll remain in git history if we ever need it again.